micro_ros_setup/config/zephyr/generic/client_uros_packages.repos

```diff
@@ -0,0 +1,38 @@
+repositories:
+  eProsima/Micro-CDR:
+    type: git
+    url: https://github.com/eProsima/micro-CDR.git
+    version: v1.1.2
+  eProsima/Micro-XRCE-DDS-Client:
+    type: git
+    url: https://github.com/eProsima/Micro-XRCE-DDS-Client.git
+    version: dashing
+
+# MicroROS
+  
+  uros/rcl:
+    type: git
+    url: https://github.com/micro-ROS/rcl
+    version: dashing
+  uros/rcutils:
+    type: git
+    url: https://github.com/micro-ROS/rcutils
+    version: feature/zephyr_fixes
+  uros/rmw_microxrcedds:
+    type: git
+    url: https://github.com/micro-ROS/rmw-microxrcedds.git
+    version: dashing
+  uros/rosidl_typesupport_microxrcedds:
+    type: git
+    url: https://github.com/micro-ROS/rosidl_typesupport_microxrcedds.git
+    version: dashing
+  uros/rosidl_typesupport:
+    type: git
+    url: https://github.com/micro-ROS/rosidl_typesupport.git
```

Comment:
I think we could use the ROS 2 repository.

Reply:
Done